topics/storybook/index.md

```diff
@@ -0,0 +1,15 @@
+---
+aliases: storybooks
+created_by: Storybook Team
```

Comment:
Specific names would be better here, or remove this field entirely, I think (seems too self-referential).

Reply:
removed it